static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js

```diff
@@ -20,14 +18,25 @@ define([
         };
 
     StickyMpu.prototype.create = function () {
-        var articleBodyOffset;
+        var element;
 
         if (this.$adSlot.data('name') !== 'right') {
             return;
         }
-        articleBodyOffset = config.page.hasShowcaseMainElement ? $('.media-primary').dim().height : qwery('.content__article-body')[0].offsetTop;
-        this.$adSlot.parent().css('height', (articleBodyOffset + mpuHeight) + 'px');
-        new Sticky(this.$adSlot[0], { top: this.opts.top }).init();
+
+        element = document.querySelector(config.page.hasShowcaseMainElement ? '.media-primary' : '.content__article-body');
```

Comment:
I would move declaration of the variable here. We had many discussions about this and we are already moving to this pattern which makes more sense with ES6 anyway.